### ISSUE TITLE
Document Keptn Project Infrastructure

### ISSUE THREAD

**Issue by oleg-nenashev** [keptn/community#93]
### Proposal

Currently we do not have much documentation about the project infrastructure used by the Keptn project. We have a bunch of services beyond GitHub, e.g. DockerHub, NPM, Google Accounts and Channels, Twitter, LinkedIn, GCP, etc. It would be nice to have them documented so that Keptn contributors could...

- [ ] Discover what services are offered by the Keptn community. E.g. I discovered NPM only after 2 months
- [ ] Discover who has access to the account
- [ ] Submit requests to get access or to have something implemented on the infrastructure
- [ ] Contribute to project infrastructure

### Suggested implementation

* A new section in https://github.com/keptn/community, with README, infrastructure map and contributing guidelines
* A new GitHub Issue template for infrastructure and permission management
* Updated Keptn contributing guidelines that reference participating in the infrastructure

### References

* Slack conversation: https://keptn.slack.com/archives/CFH319TPE/p1641303177263400
* Example: Jenkins Infrastructure sub-project: https://www.jenkins.io/projects/infrastructure

### Sponsors

- (at)sponsor-1
- (at)sponsor-2

### Checklist

- [ ] The idea is brought up for a discussion in a relevant Keptn Slack channel (use `#keptn-project` by default)
- [ ] Each sponsor should reply to this issue with the comment "*I support*".
